File: test/ocamlTests/CharTest/isLowercaseTest.ml
test "isLowercase('a')" (fun () -> expect (Char.isLowercase 'a') |> toEqual Eq.bool true) ; 
test "isLowercase('7')" (fun () -> expect (Char.isLowercase '7') |> toEqual Eq.bool false) ; 
test "isLowercase('\236')" (fun () -> expect (Char.isLowercase '\236') |> toEqual Eq.bool false) ; 

File: test/fsharpTests/CharTest/isLowercaseTest.fs
testCase "isLowercase('a')" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Char.isLowercase 'a') "error"
testCase "isLowercase('7')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLowercase '7') "error"
testCase "isLowercase('\236')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isLowercase '\236') "error"
